LOGGING/test/EventSourceListener.Tests/TraceTelemetryComparer.cs

```diff
@@ -31,6 +31,9 @@ public int Compare(object x, object y)
                 return 0;
             }
 
+            var templateHash = template.GetHashCode();
+            var actualHash = template.GetHashCode();
```

Comment:
what is this for?

Reply:
hum. my bad. forgot to remove. will do it soon.

Reply:
ok! next commit will remove those!